event-exporter/sinks/stackdriver/sink_test.go

```diff
@@ -28,14 +28,13 @@ import (
 )
 
 type fakeSdWriter struct {
-	writeFunc func([]*sd.LogEntry, string, *sd.MonitoredResource) int
+	writeFunc func([]*sd.LogEntry, string, *sd.MonitoredResource)
```

Comment:
Good catch. Although, I suspect this couldn't have even compiled previously. Is our CI system not checking this? It may be worth while to see if something is misconfigured.